pages/docs/manual/latest/interop-cheatsheet.mdx

```diff
@@ -193,6 +194,27 @@ external join: array<string> => string = "join"
 
 </CodeTab>
 
+### Tagged template functions
+
+<CodeTab labels={["ReScript", "JS Output"]}>
+
+```res example
+type res 
+@module("bun") @taggedTemplate 
```

Comment:
Same comment as above about type naming, return type, and linking to the Bun docs for context.